doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -290,7 +290,7 @@ single-locale setting:
 
 ``foo.skipif``
 
-  .. code-block::
+  .. code-block:: bash
```

Comment:
Hm our skipifs are technically not bash.

Reply:
True, but it works in this case.

Reply:
True, but it looks better with highlighting for this example.